generate_workspace/src/main/java/com/google/devtools/build/workspace/maven/BUILD

```diff
@@ -3,20 +3,17 @@ java_library(
     srcs = glob(["*.java"]),
     visibility = ["//generate_workspace/src:__subpackages__"],
     runtime_deps = [
-        "@org_codehaus_plexus_plexus_interpolation//jar",
-        "@org_codehaus_plexus_plexus_utils//jar",
+        "//third_party:org_codehaus_plexus_plexus_interpolation",
```

Comment:
I see that you have changed all of the external dependencies to use third party java_libraries, rather than the maven_jar rules. 

Does this change mean that we can remove the following lines from the workspace file
```
load('//:generate_workspace.bzl', 'generated_maven_jars')
generated_maven_jars()
```
Also, out of curiosity, why have you chosen to include the java_library targets rather than the maven_central ones?